test/arcade/potts/agent/cell/PottsCellTest.java

```diff
@@ -1179,8 +1075,6 @@ public void convert_withRegions_createsContainer() {
         assertEquals(pop, container.pop);
         assertEquals(age, container.age);
         assertEquals(divisions, container.divisions);
-        assertEquals(state, container.state);
```

Comment:
We do still want to check that state and phase are correctly set. You'll need to adjust the `PottsCellMock` module mock to actually set values based on the `state` and `phase` that get passed to the constructor

Reply:
If we are moving the module assignment to the subclasses does it make sense to check for state and phase here? This would require me implementing part of the PottsCellStem in the PottsCellMock constructor.

Reply:
The purpose of this test is to check that calling `convert` creates a container with the correct values, which includes state/phase. It doesn't matter what the state/phase actually is, or which states/phases are valid. The tests in `PottsCellStem` would be the ones that check if states/phases are set with their corresponding modules. Does that difference make sense? Happy to hop on a call if needed!